velocity-engine-core/src/main/java/org/apache/velocity/VelocityContext.java

```diff
@@ -48,7 +45,9 @@
  *  @author <a href="mailto:dlr@finemaltcoding.com">Daniel Rall</a>
  *  @version $Id$
  */
-public class VelocityContext extends AbstractContext implements Cloneable
+public class VelocityContext
+    extends AbstractContext
+    implements Cloneable, Serializable
```

Comment:
I did just because has a serialVersionUID with this comment "Version Id for serializable" and It doesn't. Eclipse throws me an error with this. If this class isn't serializable maybe I can remove the interface and the serialVersionUID.